lib/deploy_complexity/output_formatter.rb

```diff
@@ -0,0 +1,165 @@
+# frozen_string_literal: true
+
+require 'values'
+
+module DeployComplexity
+  Attachment = Value.new(:title, :text, :color)
+
+  # Parent class for formatting deploy complexity output
+  # Child classes (slack, cli) should implement #format and #format_attachment
```

Comment:
Yeah, this is definitely a pretty big blob getting formatted in here. I was sort of modeling this off of the Assignments API formatted in the monolith, which does indeed do a lot. 

I agree that this could continue to be decomposed as we continue work on the deploy-complexity output. For now my mentality was sort of like... "let's get it in a module!" 😛